### PR TITLE
Fix running untitled scripts with arguments (but break line breakpoints) 

### PR DESCRIPTION
**Pull request by andschwa**
So the extant hack that enabled debugging untitled scripts is really
untenable. It shifted the user's args by one since it ran the untitled
script as the first arg to the dot-source operator. Unfortunately, there
doesn't seem to be a clear way to break on lines in an untitled script.
However, `Wait-Debugger` and command breakpoints should still work.

Fixes https://github.com/PowerShell/vscode-powershell/issues/3655 and https://github.com/PowerShell/vscode-powershell/issues/3770

### CODE REVIEW

File: src/PowerShellEditorServices/Services/DebugAdapter/Handlers/ConfigurationDoneHandler.cs [hunk 1]
```diff
@@ -102,48 +102,36 @@ public Task<ConfigurationDoneResponse> Handle(ConfigurationDoneArguments request
...
-                    ScriptBlockAst ast = Parser.ParseInput(untitledScript.Contents, untitledScript.DocumentUri.ToString(), out Token[] tokens, out ParseError[] errors);
-
-                    // This seems to be the simplest way to invoke a script block (which contains breakpoint information) via the PowerShell API.
-                    //
-                    // TODO: Fix this so the added script doesn't show up.
-                    var cmd = new PSCommand().AddScript(". $args[0]").AddArgument(ast.GetScriptBlock());
-                    await _executionService
-                        .ExecutePSCommandAsync<object>(cmd, CancellationToken.None, s_debuggerExecutionOptions)
-                        .ConfigureAwait(false);
+                    ScriptBlockAst ast = Parser.ParseInput(
```
Comment (andschwa) [inline src/PowerShellEditorServices/Services/DebugAdapter/Handlers/ConfigurationDoneHandler.cs:116]: This whole thing is pointless if we decided to break line breakpoints for untitled files.

Comment (andschwa) [reply src/PowerShellEditorServices/Services/DebugAdapter/Handlers/ConfigurationDoneHandler.cs:116]: @SeeminglyScience Do you think we could send a `ScriptBlock` type down the pipeline executor (much like `PSCommand`) to eventually `.Invoke()` it?

**Comment by andschwa** [comment]
This essentially reverts #1611 and stops attempting to quote arguments given to the script launcher.

**Review by JustinGrote — APPROVED** [review]
LGTM. RIP argumentescapingtests 🪦

**Review by PaulHigin — APPROVED** [review]

**Comment by TylerLeonhardt** [comment]
darn :( I really liked untitled script debugging

**Comment by andschwa** [comment]
You can still debug it, just not with line breakpoints. We cannot support both that _and_ passing arguments to the script. `Wait-Debugger` in your script works, as does command breakpoints.

**Comment by ALIENQuake** [comment]
@andschwa 
This seems like an outstanding feature for day-to-day development and I don't agree that it should be removed over 'passing parameters'. Passing parameters are meant for actual files. You can also wrap script content as a function and pass parameters. You can also provide default values for parameter variables at the beginning of the file. But since this is a lost cause already, can you please add an option to restore old behavior?

Also, I reverted to the previous version of the Preview extension and I'm not able to perform 'untitled script debugging'. This is the code I'm using:

```
Get-ChildItem 'C:\'
$a = Get-Verb | select -First 3 # <- the line breakpoint
Get-item '.'
```
What I'm missing? What is the latest extension version that has this feature enabled?

**Comment by andschwa** [comment]
@ALIENQuake We are indeed looking to see if we can keep both behaviors. We don't want to keep the hack that previously supported the feature, but are trying to find a way to do it in a less hacky manner. Thing is, in my experience the line breakpoints were frequently off by one in untitled files because it was _not_ a file being passed but the parsed AST.

**Comment by andschwa** [comment]
The fix for this is out in [PowerShell Preview](https://marketplace.visualstudio.com/items?itemName=ms-vscode.powershell-preview)! Line breakpoints were re-enabled _and_ arguments are supported! Please note that the code that's run _looks_ funky ('cause it is) but it works!

**Comment by ALIENQuake** [comment]
@andschwa Wonderfull! Many thanks for your extra effort 👍🏻

**Comment by andschwa** [comment]
To be totally clear I would say in PowerShell 7+ by design (where the APIs to do this are available).

**Pull request merged**